feature-group-definitions/intersection-observer-v2.yml

```diff
@@ -3,6 +3,10 @@ description: The `trackVisibility` parameter for the `IntersectionObserver` cons
 spec: https://w3c.github.io/IntersectionObserver/v2/
 caniuse: intersectionobserver-v2
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/3051
-compat_features:
-  - api.IntersectionObserver.IntersectionObserver.options_trackVisibility_parameter
-  - api.IntersectionObserverEntry.isVisible
```

Comment:
It's weird that we thought that this was in BCD, but it wasn't. Is this something we ought to file an issue against in BCD?

Reply:
My memory of creating https://github.com/web-platform-dx/web-features/pull/628 is imperfect, but I think I just predicted BCD keys without checking if they were in BCD. I've filed https://github.com/mdn/browser-compat-data/issues/22862.